java/group-layers/src/main/java/com/esri/arcgisruntime/sample/grouplayers/MainActivity.java

```diff
@@ -90,13 +91,19 @@
     FeatureLayer projectArea = new FeatureLayer(
         new ServiceFeatureTable(getString(R.string.project_area_feature_service)));
 
-    // create a group layer from scratch by adding the layers as children
+    // create a group layer from scratch by adding the trees, pathways, and project area as children
+    GroupLayer projectAreaGroupLayer = new GroupLayer();
+    projectAreaGroupLayer.setName("Project area group");
+    projectAreaGroupLayer.getLayers().addAll(Arrays.asList(projectArea, pathways, trees));
+
+    // create a group layer for the buildings and set its visibility mode to exclusive
     GroupLayer buildingsGroupLayer = new GroupLayer();
     buildingsGroupLayer.setName("Buildings group");
     buildingsGroupLayer.getLayers().addAll(Arrays.asList(buildingsA, buildingsB));
+    buildingsGroupLayer.setVisibilityMode(GroupVisibilityMode.EXCLUSIVE);
 
     // add the group layer and other layers to the scene as operational layers
-    scene.getOperationalLayers().addAll(Arrays.asList(projectArea, buildingsGroupLayer, trees, pathways));
+    scene.getOperationalLayers().addAll(Arrays.asList(projectAreaGroupLayer, buildingsGroupLayer));
```

Comment:
good name change here, makes it much more readable

Reply:
Thanks!